play-v27/src/main/scala/com/gu/googleauth/filters.scala

```diff
@@ -12,7 +13,7 @@ object GoogleAuthFilters {
   val LOGIN_ORIGIN_KEY = "loginOriginUrl"
   class AuthFilterWithExemptions(
       loginUrl: FilterExemption,
-      exemptions: Seq[FilterExemption])(implicit val mat: Materializer, environment: Environment) extends Filter {
+      exemptions: Seq[FilterExemption])(implicit val mat: AkkaOrPekko.Materializer, environment: Environment) extends Filter {
```

Comment:
In Play 3 and later, `com.gu.googleauth.internal.AkkaOrPekko.Materializer` will be `org.apache.pekko.stream.Materializer` (thanks to the [`src-pekko`](https://github.com/guardian/play-googleauth/pull/200/files#diff-db48f773e8b655d8e69a5da70f0e76a0767bae17f19cddbe28057c7f62308745) folder), in older versions it will be `akka.stream.Materializer` (thanks to [`src-akka`](https://github.com/guardian/play-googleauth/pull/200/files#diff-7407f3cd78ceb9b628ca671ea967e36fa985dea16a2ca39bfa8921cc894b29ab)).

See `PlayVersion.pekkoOrAkkaSrcFolder`, which controls which `src-xxx` folder gets used.